scripts/pre_commit_linter.py

```diff
@@ -1085,6 +1086,102 @@ def _check_directive_scope(all_files):
     return summary_messages
 
 
+class CustomHTMLParser(HTMLParser.HTMLParser):
+    """Custom HTML parser to check indentation."""
+
+    def __init__(self, filename, failed=False):
+        HTMLParser.HTMLParser.__init__(self)
+        self.failed = failed
+        self.filename = filename
+
+    def handle_starttag(self, tag, attrs):
+        line_number, column_number = self.getpos()
+        # Check the indentation of the tag.
+        if column_number % 2 != 0 and (
```

Comment:
For some reason, this doesn't appear to trigger when I set an Html tag to the indentation at pos 3 instead of 2 (assuming 1 indexing).

Reply:
Hmm. Okay, let me look into this.

Reply:
Done. Modified the custom parser accordingly.